PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessClusterUpdateDialog.java

```diff
@@ -112,6 +113,13 @@ public void setData(@NotNull SparkServerlessClusterProvisionSettingsModel data)
         workerMemoryField.setText(String.valueOf(data.getWorkerMemory()));
         workerNumberOfContainersField.setText(String.valueOf(data.getWorkerNumberOfContainers()));
 
-        errorMessageField.setText(data.getErrorMessage());
+        if (!StringUtils.isEmpty(data.getErrorMessage())) {
+            if (!errorMessageDecorator.isExpanded()) {
+                errorMessageDecorator.setOn(true);
+            }
+
+            printLogLine(ConsoleViewContentType.ERROR_OUTPUT, data.getErrorMessage());
+            printLogLine(ConsoleViewContentType.NORMAL_OUTPUT, "Cluster guid: " + cluster.getGuid());
```

Comment:
Should the `NORMAL_OUTPUT` line be took out from only including error message condition?

Reply:
Taking the cluster guid log out of error message condition means cluster guid will also be shown when provision success. However, the cluster guid is only needed when provision failed to track the issue for backend team.

---
In reply to: [213200665](https://github.com/Microsoft/azure-tools-for-java/pull/1879#discussion_r213200665) [](ancestors = 213200665)